xmodule/video_block/video_block.py

```diff
@@ -237,11 +237,11 @@ def prioritize_hls(self, youtube_streams, html5_sources):
 
         return False
 
-    def student_view(self, _context):
+    def student_view(self, context):
```

Comment:
Does this variable name change the behavior of the function?

Reply:
Actually it doesn't change anything, but I think the convention of using `_` before a variable name is to point out that the variable is not being used in its scope. That's why I removed that `_`

Reply:
Is it necessary to use `context` now? I saw that you updated `self.get_html()` to use `context`, but is that necessary?

Reply:
Since `dest_lang` is coming in that `context` on each render we need to start passing this value to inner functions to perform the update on the CC. Don't know if this is a correct pattern or if there are other implications to take into account. Maybe there's another way of getting the `context` to be used in https://github.com/openedx/edx-platform/pull/34667/files#diff-a7e3c5a7f8c78b1241f3617e180bc1976ac83b45e804055e4a677eed0bca87e4R859

Reply:
Your change makes sense to me! I just want to make sure that my understanding was correct. Thank you for your explanation.